api/src/opentrons/protocol_api/core/engine/module_core.py

```diff
@@ -78,6 +78,10 @@ def get_deck_slot(self) -> DeckSlotName:
         """Get the module's deck slot."""
         return self._engine_client.state.modules.get_location(self.module_id).slotName
 
+    def get_display_name(self) -> str:
+        """Get the module's display name."""
+        return f"{self._engine_client.state.modules.get_definition(self.module_id).displayName} on {self.get_deck_slot()}"
```

Comment:
Can we shuffle some stuff around a little between here and the `ModuleContext` for clarity?

```suggestion
        return self._engine_client.state.modules.get_definition(self.module_id).displayName
```